requirements.txt

```
@@ -6,9 +6,9 @@ configparser
 cwltool>=1.0.20191225192155
 docutils
 ephemeris>=0.10.3
-galaxy-containers
-galaxy-tool-util>=21.1.0.dev4
-galaxy-util>=20.5.0
+galaxy-containers<22.01
+galaxy-tool-util<22.01,>=21.1.0.dev4
+galaxy-util<22.01,>=20.5.0
```

```suggestion
galaxy-tool-util>=21.1.0.dev4,<22.01
galaxy-util>=20.5.0,<22.01
```

I think this is the "standard" way to write the pins.

---

That's what I did first, but pip always lists it the other way round (in the pip install output), so I thought that is the standard ...